### DISCUSSION TITLE
Passing a detectOverflow boundary, react version?

### DISCUSSION THREAD

**Discussion by WilliamIPark** [floating-ui/floating-ui#1767] [React DOM] [answered]
Hey! I'm just wondering is there any examples of using `detectOverflow` with `useFloating`, as I need to pass a custom boundary, but just not sure of the path to do this?

**Comment by WilliamIPark** [comment, marked as answer]
Figured this out. The options for detectOverflow go into the options for other plugins such as flip.

**Reply by Grandizer** [reply]
Could you show code, please?

**Reply by atomiks** [reply]
```js
const [boundary, setBoundary] = useState(null);

// in the middleware array
flip({ boundary: boundary || 'clippingAncestors' })

return <div ref={setBoundary} />
```
